recipes/use_cases/end2end-recipes/raft/README.md

```diff
@@ -0,0 +1,241 @@
+
+## Introduction:
+As the popularity of our Meta Llama 3 models grows, we've seen a surge in demand to adapt them to specific domains, enabling businesses to better serve their customers. For instance, a company might have a vast collection of plain text documents related to their custom domain and want to create a chatbot that can answer client questions.
+
+In response to this demand, we're exploring the possibility of building a Llama chatbot that can answer Llama related questions using our Meta Llama 3 models. In this tutorial, we'll demonstrate how to do just that. While our Meta Llama 3 70B Instruct model is an excellent candidate, as it already has a excellent reasoning capabilities and knowledge, its production costs are relatively high. To reduce these costs, we'll focus on creating a Llama chatbot based on the Meta Llama 8B Instruct model, aiming to achieve similar accuracy to the Meta Llama 3 70B Instruct model while minimizing inference costs.
+
```

Comment:
Pls add a section on the overall plan

How to develop a chatbot:

1. RAG : We are going to use RAG and why
2. Fine-tuning: Why we will need to FT the model
3. RAFT: how it helps

Reply:
@wukaixingxp  lets add the above to the readme.

Reply:
added this discussion in the first section